src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -482,21 +482,14 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     void ConptyConnection::_LastConPtyClientDisconnected() noexcept
     try
     {
-        DWORD exitCode = 0;
+        DWORD exitCode{ 0 };
         GetExitCodeProcess(_piClient.hProcess, &exitCode);
 
         // Signal the closing or failure of the process.
         // exitCode might be STILL_ACTIVE if a client has called FreeConsole() and
         // thus caused the tab to close, even though the CLI app is still running.
-        const auto success = exitCode == 0 || exitCode == STILL_ACTIVE || exitCode == STATUS_CONTROL_C_EXIT;
```

Comment:
honestly I was actually fine w/ this change - I just thought the exitcode one was weird! sorry!!

Reply:
Nah, we can do such changes later, when they're necessary. 🙂